package.json

```diff
@@ -0,0 +1,14 @@
+{
+	"dependencies": {
```

Comment:
check indentation; would be good to add linter early to enforce styling rules

Reply:
strange, looks normal in my code editor, but ok